### PR TITLE
feat(s3_eye): Add option to change LVGL buffer location from MENUCONFIG (BSP-589)

### PR DESCRIPTION
**Pull request by nikhil-robinson**
# ESP-BSP Pull Request checklist

> Note: For new BSPs create a PR with this [link](https://github.com/espressif/esp-bsp/compare/main...my-branch?quick_pull=1&template=pr_template_bsp.md).
- [x] Version of modified component bumped
- [x] CI passing

# Change description
This pull request introduces an option in menuconfig to change the LVGL buffer location from DMA to SPIRAM for the ESP32-S3-EYE.

### CODE REVIEW

**Review by espzav — APPROVED** [review]
Thank you for this PR. Left only one suggestion. LGTM

**Comment by nikhil-robinson** [comment]
@espzav The commits i made was not linked to my email id. Signing is not passing. 😢

**Comment by espzav** [comment]
> @espzav The commits i made was not linked to my email id. Signing is not passing. 😢

@tore-espressif Please could you help with this? Is it ok to merge without it?

File: bsp/esp32_s3_eye/Kconfig [hunk 1]
```diff
@@ -86,6 +86,12 @@ menu "Board Support Package"
             range 1 500
             help
                 Maximum time for task sleep in ms.
+        
+        config BSP_DISPLAY_LVGL_BUFFER_IN_PSRAM
+            bool "Allocate LVGL buffer in PSRAM "
+            default n
+            help
+                LVGL buffer will be Allocated in PSRAM instead of DMA.
```
[the remark this answers is not in the document]
Comment (tore-espressif) [reply bsp/esp32_s3_eye/Kconfig:94]: @nikhil-robinson can you please resolve the conversation only after the change is pushed?

**Review by tore-espressif — APPROVED** [review]
> > @espzav The commits i made was not linked to my email id. Signing is not passing. 😢
> 
> @tore-espressif Please could you help with this? Is it ok to merge without it?

Yes, we can merge like this, LGTM!

File: bsp/esp32_s3_eye/Kconfig (hunk 1 above)
[the remark this answers is not in the document]
Comment (nikhil-robinson) [reply bsp/esp32_s3_eye/Kconfig]: Done !!

**Pull request merged**